pkg/backup/backupaction/rbac.go

```diff
@@ -14,7 +14,7 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package backup
```

Comment:
Should this file be moved to its own package? It doesn't seem like it fits in `backupaction` to me, even if some actions depend on it.